packages/client-python/src/rocketride/mixins/store.py

```
@@ -21,868 +21,353 @@
+        for entry in json_entries:
+            try:
+                item_id = entry['name'][:-5]
+                pipeline = await self.fs_read_json(f'{directory}/{entry["name"]}')
+                sources = []
+                for component in pipeline.get('components', []):
+                    config = component.get('config', {})
+                    if config.get('mode') == 'Source':
+                        sources.append({'id': component.get('id'), 'provider': component.get('provider'), 'name': config.get('name', component.get('id'))})
+                items.append({id_key: item_id, 'name': pipeline.get('name', 'Untitled'), 'description': pipeline.get('description', ''), 'sources': sources, 'totalComponents': len(pipeline.get('components', []))})
+            except Exception:
+                continue
```

Ruff (0.15.7)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)